### PR TITLE
Add Logging to AuthenticationService Scripts

### LINKED ISSUE #45234: [Blazor] Add missing logging
image

Seems that we missed logging in a couple of cases like the exception depicted in the referenced issue, we should add the log statements to help troubleshoot what is going on.

_Originally posted by @kbeaugrand in https://github.com/dotnet/aspnetcore/issues/45187#issuecomment-1322572201_

### PR DESCRIPTION
**Pull request by Nick-Stanton**
# Add Logging to AuthenticationService Scripts

Fills out Blazor WASM authentication scripts with more logging statements to debug customer issues easier.

cc @javiercn
Fixes #45234

### CODE REVIEW

File: src/Components/WebAssembly/WebAssembly.Authentication/src/Interop/AuthenticationService.ts [hunk 1]
```diff
@@ -296,7 +302,9 @@ class OidcAuthorizeService implements AuthorizeService {
                 return this.operationCompleted();
             }
         } catch (error) {
-            return this.error(this.getExceptionMessage(error));
+            const message = this.getExceptionMessage(error);
+            this.debug(`Complete sign out error '${message}'`);
+            return this.error(error);
```
Comment (TanayParikh) [inline src/Components/WebAssembly/WebAssembly.Authentication/src/Interop/AuthenticationService.ts:307]: I believe this should be the message?

```suggestion
            return this.error(message);
```

**Review by TanayParikh — COMMENTED** [review]
LGTM, will await Javier for final signoff as he has more context in this area.

File: src/Components/WebAssembly/WebAssembly.Authentication/src/Interop/AuthenticationService.ts (hunk 1 above)
Comment (Nick-Stanton) [reply src/Components/WebAssembly/WebAssembly.Authentication/src/Interop/AuthenticationService.ts:307]: Woops, great catch

**Review by javiercn — APPROVED** [review]
Thanks for taking care of it!

Seems like I missed a couple of spots 😄

**Comment by Nick-Stanton** [comment]
Test failure tracked by #45378 and is unrelated to these changes.

**Comment by TanayParikh** [comment]
If the re-run doesn't work, an `\azp run` should do the trick (it creates a new merge commit - and this test is now quarantined in main).

**Pull request merged**